winui/Kanban/column.md

```diff
@@ -0,0 +1,172 @@
+
+### Populating columns with cards from Multiple categories
+
+You can populate a single column with cards from multiple categories by assigning multiple values to the `Categories` collection of the `KanbanColumn`. For example, to group tasks categorized as `Done` and `Validated` under a single `Done`column, you can include both category values in the `Categories` collection.
+
+{% tabs %}
+{% highlight XAML hl_lines="2 7" %}
+
+<kanban:SfKanban x:Name="kanban"
+                 AutoGenerateColumns="False" 
+                 ItemsSource="{Binding TaskDetails}">
+        <kanban:KanbanColumn HeaderText="To Do" Categories="Open" />
+        <kanban:KanbanColumn HeaderText="In Progress" Categories="In Progress" />
+        <kanban:KanbanColumn HeaderText="Code Review" Categories="Code Review" />
+        <kanban:KanbanColumn HeaderText="Done" Categories="Done, Validated" />
+</kanban:SfKanban>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="4" %}
+
+this.kanban.Columns.Add(new KanbanColumn() { HeaderText = "To Do", Categories = "Open" });
+this.kanban.Columns.Add(new KanbanColumn() { HeaderText = "In Progress", Categories = "In Progress" });
+this.kanban.Columns.Add(new KanbanColumn() { HeaderText = "Code Review", Categories = "Code Review" });
+this.kanban.Columns.Add(new KanbanColumn() { HeaderText = "Done", Categories = "Done, Validated" });
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customizing column headers
+
+### Customizing column header text
+
+You can set the header text for a column using the `HeaderText` property, which allows you to display simple text as the column title.
+
+{% tabs %}
+{% highlight XAML %}
+
+<kanban:KanbanColumn Categories="Open" HeaderText="To Do"/>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+this.kanban.Columns.Add(new KanbanColumn() { Categories = "Open", HeaderText = "To Do" });
+
+{% endhighlight %}
+{% endtabs %}
+
+### Customizing column header template
+
+To replace the UI of the header, you can use the `ColumnHeaderTemplate` of the `SfKanban`. The following code example illustrates how this can be done.
+
+{% tabs %}
+{% highlight XAML %}
+
+<kanban:SfKanban x:Name="kanban"
+                 ItemsSource="{Binding TaskDetails}">
+    <kanban:SfKanban.ColumnHeaderTemplate>
+        <DataTemplate>
+            <StackPanel Background="LightBlue">
+                <TextBlock Margin="10" Text="{Binding  HeaderText}" Foreground="Red" HorizontalAlignment="Center"/>
+            </StackPanel>
+        </DataTemplate>
+    </kanban:SfKanban.ColumnHeaderTemplate>
+    <kanban:SfKanban.DataContext>
+        <local:ViewModel/>
+    </kanban:SfKanban.DataContext>
+</kanban:SfKanban>
+
+{% endhighlight %}
+{% endtabs %}
+
+## Expanding or collapsing column
+
+To expand or collapse a column programmatically, use the `KanbanColumn.IsExpanded` property. Set this property to `true` to expand the column or `false` to collapse it.
```

Comment:
The Kanban allows you to programmatically expand or collapse columns using the `IsExpanded` property of `KanbanColumn` in the `SfKanban` control. By default, columns can be expanded or collapsed by tapping the toggle button placed at the top-right corner of the kanban column header.

Reply:
Content changed